drivers/usb/udc/udc_dwc2_vendor_quirks.h

```diff
@@ -108,7 +108,7 @@ DT_INST_FOREACH_STATUS_OKAY(QUIRK_STM32F4_FSOTG_DEFINE)
 
 #endif /*DT_HAS_COMPAT_STATUS_OKAY(st_stm32f4_fsotg) */
 
-#if DT_HAS_COMPAT_STATUS_OKAY(nordic_nrf_usbhs)
+#if DT_HAS_COMPAT_STATUS_OKAY(nordic_nrf_usbhs_nrfs)
```

Comment:
There is no point in renaming it or postfixing it with -nrfs. And there is no reason to break someone else's work with a compatible rename. You will need to communicate with us internally to find a suitable compatible.

Reply:
There is a direct need for solving the compatible naming issue. Both nRF54H20 and nRF54L20 do have USBHS peripheral. Yet, the nRF54H20 requires nrfs to get information about VBUS and nRF54L20 has USBREG peripheral for that. Therefore I think the `nordic,nrf-usbhs-nrfs` and `nordic,nrf-usbhs-usbreg` is probably the cleanest approach.

Reply:
I have no strong opinion, although I think the `-nrfs` suffix makes the difference clear.

Reply:
I am not convinced to approve changes that break other people's work for no reason. "nordic, nrf-usbhs" should not be renamed. Also, new compatilbe needs a better name, something like "nordic, nrf-usbhs-fancy-soc-family". (nrfs is some software, usbreg is ambiguous)

Reply:
I have removed the `nordic,nrf-usbhs` change and the nRF54L20 now has `nordic,nrf-usbhs-nrf54l` compatible instead.